roda-core/roda-core/src/main/java/org/roda/core/storage/DefaultTransactionalStorageService.java

```diff
@@ -604,22 +601,34 @@ public void importBinaryVersion(StorageService fromService, StoragePath storageP
 
   @Override
   public void commit() throws RODATransactionException {
+    Map<StoragePathVersion, List<ConsolidatedOperation>> consolidatedOperations;
```

Comment:
The consolidated logs were supposed to be written directly to a table to avoid keeping them in memory. Also, a new column was expected to store the commit state. We shouldn’t be changing the status of the operation that was already performed.